boofuzz/fuzz_logger.py

```diff
@@ -70,6 +70,14 @@ def log_send(self, data):
         for fuzz_logger in self._fuzz_loggers:
             fuzz_logger.log_send(data=data)
 
+    def end_test_case(self):
+        for fuzz_logger in self._fuzz_loggers:
+            fuzz_logger.end_test_case()
+
+    def end_test(self):
+        for fuzz_logger in self._fuzz_loggers:
+            fuzz_logger.end_test()
+
```

Comment:
Oops, that probably should have been "close_test_case" and "close_test" to match better with open. 🤷‍♂️ Feel free to change if you agree but not required.